src/shell/lisa_shell

```diff
@@ -335,24 +346,37 @@ while [[ $# -gt 0 ]]; do
                         return 0
                         ;;
                 -n|--iterations)
-                        iterations="$2"
+                        LISA_TEST_ITERATIONS="$2"
+                        shift
+                        ;;
+
+                -t|--target-conf)
+                        LISA_TARGET_CONF="$2"
+                        shift
+                        ;;
+                -o|--results-dir)
+                        LISA_RESULTS_DIR="$2"
                         shift
                         ;;
                 *)
                         # Unrecognised args are passed through to nosetests
-                        extra_args="$extra_args $1"
+						extra_args+=("$1")
```

Comment:
That's more than one whitespace too far :p

Reply:
Ah, accidental conversion of spaces to tabs